protocol/scripts/deploys/v2/sepolia/99-post-deployment.ts

```diff
@@ -0,0 +1,123 @@
+import { ethers } from 'hardhat';
+import {
+    ensureExpectedEnvvars,
+    mine,
+} from '../../helpers';
+import { connectToContracts, getDeployedContracts } from '../sepolia/contract-addresses';
+import { keccak256 } from 'ethereumjs-util';
+import { ITempleStrategy } from '../../../../typechain';
+
+async function main() {
+    ensureExpectedEnvvars();
+  
+    const [owner] = await ethers.getSigners();
+    const TEMPLE_V2_DEPLOYED = getDeployedContracts();
+    const templeV2contracts = connectToContracts(owner);
+    const EXTERNAL_ALL_USERS  = keccak256(Buffer.from("EXTERNAL_USER"));
+
+    // Setup Circuit Breaker for all users
+    {
+        await mine(templeV2contracts.circuitBreakerProxy.setIdentifierForCaller(owner.getAddress(), "EXTERNAL_USER"));
+
+        // Circuit breaker for Temple
+        await mine(templeV2contracts.circuitBreakerProxy.setCircuitBreaker(
+            EXTERNAL_ALL_USERS,
+            TEMPLE_V2_DEPLOYED.CORE.TEMPLE_TOKEN,
+            TEMPLE_V2_DEPLOYED.CORE.CIRCUIT_BREAKER_PROXY
```

Comment:
The 3rd param isn't right. What you have now is it pointing to itself.

It should be `TEMPLE_V2_DEPLOYED.TEMPLE_LINE_OF_CREDIT.CIRCUIT_BREAKERS.TEMPLE`